js/ladda.js

```diff
@@ -349,9 +349,15 @@
 			spinnerColor;
 
 		if( height === 0 ) {
-			// We may have an element that is not visible so
-			// we attempt to get the height in a different way
-			height = parseFloat( window.getComputedStyle( button ).height );
+			//Make sure this function working with client browser
+			if (typeof(window.getComputedStyle) != "undefined"){
+				// We may have an element that is not visible so
+				// we attempt to get the height in a different way
+				height = parseFloat( window.getComputedStyle( button ).height );
+				
+				if (isNaN(height))
+					height = 0;
+			}
```

Comment:
Can you add spaces inside brackets to be consistent in style with other file, please?